ketcher-autotests/tests/File-Management/Open-And-Save-Files/Rxn-Files/rxn-files.spec.ts

```diff
@@ -1,12 +1,278 @@
+// Need to create new bug looks like it connected to #2389 issue
+test.fixme(
+  'Open and Save file - Reaction from file that contains Sgroup',
+  async ({ page }) => {
+    /**
+     * Test case: EPMLSOPKET-1903
+     * Description: Reaction from file that contains Sgroup
+     */
+    await page.goto('');
+    await openFileAndAddToCanvas(
+      'RXN/structure-with-s-groups-with-unsupported-s-group-type.rxn',
+      page,
+    );
+    const expectedFile = await getRxn(page);
+    await saveToFile(
+      'RXN/structure-with-s-groups-with-unsupported-s-group-type-saved.rxn',
+      expectedFile,
+    );
+    await openFileAndAddToCanvas(
+      'RXN/structure-with-s-groups-with-unsupported-s-group-type-saved.rxn',
+      page,
+    );
+    await takeEditorScreenshot(page);
+  },
+);
+
+test('Open and Save file - File without arrow or(and) plus-symbol', async ({
+  page,
+}) => {
+  /**
+   * Test case: EPMLSOPKET-1905
+   * Description: File without arrow or(and) plus-symbol
+   */
+  await page.goto('');
+  await selectLeftPanelButton(LeftPanelButton.Chain, page);
+  await moveMouseToTheMiddleOfTheScreen(page);
+  const { x, y } = await getCoordinatesOfTheMiddleOfTheScreen(page);
+  const xDelta = 300;
+  const xDeltaHalf = 150;
+  const yDelta50 = 50;
+  const yDelta20 = 20;
+  const xCoordinatesWithShift = x + xDelta;
+  const xCoordinatesWithShiftHalf = x + xDeltaHalf;
+  const yCoordinatesWithShift = y + yDelta50;
+  await dragMouseTo(xCoordinatesWithShift, y, page);
+  await savedFileInfoStartsWithRxn(page);
+
+  await pressButton(page, 'Cancel');
+  await selectLeftPanelButton(LeftPanelButton.ReactionPlusTool, page);
+  await page.mouse.click(xCoordinatesWithShiftHalf, yCoordinatesWithShift);
+  const ySecondChain = yCoordinatesWithShift + yDelta50;
+  await selectLeftPanelButton(LeftPanelButton.Chain, page);
+  await page.mouse.move(x, ySecondChain);
+  await dragMouseTo(xCoordinatesWithShift, ySecondChain, page);
+  await savedFileInfoStartsWithRxn(page);
+
+  await pressButton(page, 'Cancel');
+  await selectLeftPanelButton(LeftPanelButton.Erase, page);
+  await page.mouse.click(xCoordinatesWithShiftHalf, yCoordinatesWithShift);
+  await selectNestedTool(page, ArrowTool.ARROW_FILLED_BOW);
+  const yArrowStart = y + yDelta20;
+  const yArrowEnd = yArrowStart + yDelta20;
+  await page.mouse.move(xCoordinatesWithShiftHalf, yArrowStart);
+  await dragMouseTo(xCoordinatesWithShiftHalf, yArrowEnd, page);
+  await savedFileInfoStartsWithRxn(page, true);
+
+  await pressButton(page, 'Cancel');
+  await selectTopPanelButton(TopPanelButton.Clear, page);
+  await selectNestedTool(page, ArrowTool.ARROW_FILLED_BOW);
+  await page.mouse.move(xCoordinatesWithShiftHalf, yArrowStart);
+  await dragMouseTo(xCoordinatesWithShiftHalf, yArrowEnd, page);
+  await savedFileInfoStartsWithRxn(page, true);
+});
+
+// EPMLSOPKET-8904 - TO DO
+// TO DO: looks like issue was not fixed
+// Need to recheck issue #1837, added new screenshot there
```

Comment:
Please discuss next steps with @Zhirnoff . If issue is not fixed we need to reopen it and fix